Sources/StreamChatTestTools/CustomUIViewSubclasses.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  CustomUIViewSubclasses.swift
+//  Sample
+//
+//  Created by Dominik Bucher on 05.03.2021.
+//  Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
```

Comment:
I don't think your fix for formatter works

Reply:
Clearly something broken on my side :trollface: Will take a look what is broken on my side... But there is possibility that adding new file while amending breaks something too because of the handling in `git add`... Now I will create ticket if I haven't :/